RetailCoder.VBE/Settings/GeneralConfigProvider.cs

```diff
@@ -68,5 +68,15 @@ protected virtual void OnAutoSaveSettingsChanged(EventArgs e)
                 handler(this, e);
             }
         }
+
+        public event EventHandler SourceControlEnabledChanged;
+        protected virtual void OnSourceControlEnabledChanged(EventArgs e)
+        {
+            var handler = SourceControlEnabledChanged;
+            if (handler != null)
+            {
+                handler(this, e);
+            }
```

Comment:
Can we use the newer syntax here?

SourceControlEnabledChanged?.Invoke(this, e);